config/utils.js

```diff
@@ -3,5 +3,6 @@
 module.exports = {
   isPresent: variable => Boolean(variable),
   asBoolean: variable => ['true', '1'].includes(variable),
-  asInteger: variable => parseInt(variable, 10)
+  asInteger: variable => parseInt(variable, 10),
+  asArray: variable => (Array.isArray(variable) ? variable.split(',') : [])
```

Comment:
Is it necessary since it is not used in this PR?

Reply:
Necessary no. But! it might be useful for a future project, it’s only one line, so…